hack/util.sh

```diff
@@ -593,16 +598,16 @@ function install_router {
         # Create a TLS certificate for the router
         if [[ -n "${CREATE_ROUTER_CERT:-}" ]]; then
             echo "[INFO] Generating router TLS certificate"
-            oadm ca create-server-cert --signer-cert=${MASTER_CONFIG_DIR}/ca.crt \
-                 --signer-key=${MASTER_CONFIG_DIR}/ca.key \
-                 --signer-serial=${MASTER_CONFIG_DIR}/ca.serial.txt \
+            oadm ca create-server-cert --signer-cert="${MASTER_CONFIG_DIR}/ca.crt" \
+                 --signer-key="${MASTER_CONFIG_DIR}/ca.key" \
+                 --signer-serial="${MASTER_CONFIG_DIR}/ca.serial.txt" \
                  --hostnames="*.${API_HOST}.xip.io" \
-                 --cert=${MASTER_CONFIG_DIR}/router.crt --key=${MASTER_CONFIG_DIR}/router.key
-            cat ${MASTER_CONFIG_DIR}/router.crt ${MASTER_CONFIG_DIR}/router.key \
-                ${MASTER_CONFIG_DIR}/ca.crt > ${MASTER_CONFIG_DIR}/router.pem
+                 --cert="${MASTER_CONFIG_DIR}/router.crt" --key="${MASTER_CONFIG_DIR}/router.key"
+            cat "${MASTER_CONFIG_DIR}/router.crt" "${MASTER_CONFIG_DIR}/router.key" \
+                "${MASTER_CONFIG_DIR}/ca.crt" > "${MASTER_CONFIG_DIR}/router.pem"
             ROUTER_DEFAULT_CERT="--default-cert=${MASTER_CONFIG_DIR}/router.pem"
         fi
-        openshift admin router --create --credentials="${MASTER_CONFIG_DIR}/openshift-router.kubeconfig" --config="${ADMIN_KUBECONFIG}" --images="${USE_IMAGES}" --service-account=router ${ROUTER_DEFAULT_CERT-}
+        openshift admin router --create --credentials="${MASTER_CONFIG_DIR}/openshift-router.kubeconfig" --config="${ADMIN_KUBECONFIG}" --images="${USE_IMAGES}" --service-account=router "${ROUTER_DEFAULT_CERT-}"
```

Comment:
If `$ROUTER_DEFAULT_CERT` is empty, do you want to give the command an empty paramater?